src/platform/site-wide/header/components/MenuItemLevel1/index.js

```diff
@@ -0,0 +1,133 @@
+    return null;
+  }
+
+  // Format menuSections to be an array if needed.
+  if (item?.menuSections) {
+    item.menuSections = formatMenuItems(item.menuSections); // eslint-disable-line no-param-reassign
+  }
+
+  // Derive the menu item's ID.
+  const menuItemID = deriveMenuItemID(item, '1');
+
+  // Derive if we the menu item is expanded.
+  const isExpanded = menuItemID === expandedMenuID;
+
+  const toggleShowItems = () => {
+    // Update the expanded menu ID.
+    updateExpandedMenuID(isExpanded ? undefined : menuItemID);
+
+    // Derive the header element.
+    const header = document.querySelector('header');
+
+    // Scroll to the top of the header when toggling a menu item.
+    if (header) {
+      header.scrollIntoView();
+    }
+  };
+
+  return (
+    <li
+      className="vads-u-background-color--primary-darker vads-u-margin--0 vads-u-margin-bottom--0p5 vads-u-width--full vads-u-font-weight--bold"
+      role="menuitem"
+    >
+      {/* Raw title */}
+      {!item?.menuSections &&
+        !item?.href && (
+          <span className="vads-u-display--flex vads-u-margin--0 vads-u-padding--2 vads-u-color--white vads-u-width--full">
+            {item?.title}
+          </span>
+        )}
+
+      {/* Title link */}
+      {!item?.menuSections &&
+        item?.href && (
+          <a
+            className="vads-u-display--flex vads-u-text-decoration--none vads-u-margin--0 vads-u-padding--2 vads-u-color--white vads-u-width--full"
+            href={item?.href}
+          >
+            {item?.title}
+          </a>
+        )}
+
+      {item?.menuSections && (
+        <>
+          {/* Expand title */}
+          <button
+            aria-expanded={isExpanded ? 'true' : 'false'}
+            className="header-menu-item-button vads-u-background-color--primary-darker vads-u-display--flex vads-u-justify-content--space-between vads-u-width--full vads-u-text-decoration--none vads-u-margin--0 vads-u-padding--2 vads-u-color--white"
+            id={menuItemID}
+            onKeyDown={event => event.keyCode === 13 && toggleShowItems()}
+            onMouseUp={toggleShowItems}
+            type="button"
+          >
+            {item?.title}
+            {isExpanded ? (
+              <i
+                aria-hidden="true"
+                className="fa fa-minus vads-u-margin-left--1 vads-u-font-size--lg"
+              />
+            ) : (
+              <i
+                aria-hidden="true"
+                className="fa fa-plus vads-u-margin-left--1 vads-u-font-size--lg"
+              />
+            )}
+          </button>
+
+          {/* Level 2 menu items */}
+          {isExpanded && (
+            <ul
+              aria-label={item?.title}
+              className="vads-u-background-color--gray-lightest vads-u-display--flex vads-u-flex-direction--column usa-unstyled-list vads-u-margin--0 vads-u-padding--0"
+              id={menuItemID}
+              role="menu"
```

Comment:
Can remove role=menu here as it doesn't apply to navigation and implies arrow-key movement